add basic error message on fatal PHP error

johndoh opened the pull request:
rather than showing a blank screen on a fatal PHP error this adds a simple error message:
![Untitled-1](https://user-images.githubusercontent.com/88682/81494074-c45a6280-929d-11ea-9e70-2c5971187c00.jpg)
this will catch common errors like typos in the config file which cause a parse error or missing PHP extensions such as Mbstring and give admins a hint to check the log files for info.

johndoh committed:
add basic error message on fatal PHP error

Show a basic error message rather than a blank screen on fatal PHP error

alecpl commented:
I wouldn't use "Roundcube" word on that page. How about "For administrators: Please, check the application and/or server error log for more information."?

johndoh commented:
Good point. I've changed the message and squashed the commits.

Note: I left the Roundcube name in the CLI error message because only admins will see that,
